platform/src/main/java/org/stellar/anchor/platform/component/platform/RpcActionBeans.java

```diff
@@ -165,18 +200,26 @@ NotifyOffchainFundsReceivedHandler notifyOffchainFundsReceivedHandler(
 
   @Bean
   NotifyOffchainFundsSentHandler notifyOffchainFundsSentHandler(
+      Sep6TransactionStore txn6Store,
```

Comment:
We def should group this services together to simplify all factory methods for RPC handlers

Reply:
Yeah, this was quite annoying...